source/iface/queue.rst

```diff
@@ -168,10 +222,16 @@ queue_ for details.
 
 ::
 
-  template <typename T>
-  event submit(T cgf);
-  template <typename T>
-  event submit(T cgf, const queue &secondaryQueue);
+  template <typename T> event submit(T cgf);
+
+  template <typename T> event submit(T cgf, queue& secondaryQueue);
+
+Submit a command group function object to the queue, in order to be scheduled
+for execution on the device. In the context of the second overload,
+if a kernel error occurs, this command group function object is then
+scheduled for execution on the secondary queue. Returns an event,
+which corresponds to the queue the command group function object
+is being enqueued on.
 
 .. rubric:: Template parameters
```

Comment:
This section "Template parameters" doesn't have any information.  Personally, I don't think it's necessary, so I think we could just remove this section.  However, if you want to keep it, you should add some description about the `T` template parameter.